### PR TITLE
Fixed User Registration

### PR DESCRIPTION
**Pull request by oetago**
Session is now created in Application
Changed signIn method to QBUsers.signIn

### CODE REVIEW

File: brewchat/src/main/java/com/example/brewchat/Application.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ public void onCreate() {
                 getString(R.string.app_id),
                 getString(R.string.auth_key),
                 getString(R.string.auth_secret));
+        chatService.createSession();
```
Comment (arreche) [inline brewchat/src/main/java/com/example/brewchat/Application.java]: Happens when you lose the session? Where should It be created again if you loose the Internet connection?

Comment (oetago) [inline brewchat/src/main/java/com/example/brewchat/Application.java]: Do you lose the session if you lose internet connection?

Comment (arreche) [inline brewchat/src/main/java/com/example/brewchat/Application.java]: I didn't ask the right question... I think that QB server gives you a token when you open a session and this token should be renewed at some point.

So why not to try to create a session every time that the user register or login?

**Comment by arreche** [comment]
@Ryanm14 and I saw that QB has two types of sessions.

See:
http://quickblox.com/developers/Android#A_couple_of_words_about_Authentication_and_Authorization

**Comment by jephillips** [comment]
+1

**Comment by arreche** [comment]
:+1:

**Pull request merged**